Library/ViewModels/BackerDashboardViewModel.swift

```diff
@@ -106,18 +122,25 @@ public final class BackerDashboardViewModel: BackerDashboardViewModelType, Backe
       Strings.projects_count_newline_saved(projects_count: user.stats.starredProjectsCount ?? 0)
     }
 
+    let swipedToTab = self.willTransitionToPageProperty.signal
+      .takeWhen(self.pageTransitionCompletedProperty.signal.filter(isTrue))
+      .map { BackerDashboardTab.allTabs[$0] }
+
     self.navigateToTab = Signal.merge(
+      swipedToTab,
       self.backedProjectsButtonTappedProperty.signal.mapConst(.backed),
       self.savedProjectsButtonTappedProperty.signal.mapConst(.saved)
     )
 
+    self.currentSelectedTabProperty <~ self.navigateToTab
+
     self.setSelectedButton = Signal.merge(
-      self.backedButtonTitleText.skip(first: 1).mapConst(.backed),
+      self.backedButtonTitleText.skip(first: 1).mapConst(.backed).take(first: 1),
```

Comment:
For clarity, let me explain this line: 
We prefix the user fetch with the current user, which doesn't contain the number of backed projects (hi graphql!), so we have to skip the first emission and wait for the api to return with the full user data. I am adding the `.take(first: 1)` because when the user leaves this screen, we want to keep their place. In the current version, this is a bug - it resets the tab and indicator view to always select "backed".